src/test/java/org/openrewrite/java/migrate/apache/commons/lang/ApacheCommonsStringUtilsTest.java

```diff
@@ -0,0 +1,188 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.apache.commons.lang;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+@SuppressWarnings({"UnnecessaryCallToStringValueOf", "ConstantValue", "UnusedAssignment", "DataFlowIssue", "StringOperationCanBeSimplified"})
+class ApacheCommonsStringUtilsTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.parser(JavaParser.fromJavaVersion().classpath("commons-lang3"))
+          .recipe(new ApacheCommonsStringUtilsRecipes());
+    }
+
+    @Test
+    @DocumentExample
+    void ubertest() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.apache.commons.lang3.StringUtils;
+                            
+              class Foo {
+                  void test() {
+                      String in = "foo";
+                      String outString = StringUtils.defaultString(in);
+                      boolean outBool = StringUtils.isEmpty(in);
+                      String[] outArray = StringUtils.split(in);
+                      outArray = StringUtils.split(in, ", ");
+                      outBool = StringUtils.equals(in, "string");
+                      outString = StringUtils.chop(in);
+                      outString = StringUtils.replace(in, "o", "z");
+                      outString = StringUtils.strip(in);
+                      outString = StringUtils.join(in);
+                      outString = StringUtils.deleteWhitespace(in);
+                      outString = StringUtils.abbreviate(in, 5);
+                      outString = StringUtils.trimToEmpty(in);
+                      outString = StringUtils.substringAfter(in, ",");
+                      outString = StringUtils.right(in, 5);
+                      outString = StringUtils.mid(in, 2, 4);
+                      outString = StringUtils.stripStart(in, "chars");
+                      outString = StringUtils.uncapitalize(in);
+                      outString = StringUtils.capitalize(in);
+                      outString = StringUtils.removeEnd(in, "remove");
+                      int outInt = StringUtils.countMatches(in, "pattern");
+                      outString = StringUtils.trimToNull(in);
+                      outInt = StringUtils.indexOfAny(in, "search");
+                      outString = StringUtils.swapCase(in);
+                      outBool = StringUtils.isAlpha(in);
+                      outBool = StringUtils.isAlphaSpace(in);
```

Comment:
I'd sort these based on the method called as well, such that it's again easier to spot what might not be covered by tests yet.